### ISSUE TITLE
Connect-HPOVMGMT should display the login message if Acknowledgement is set to $true ### Expected Behavior

### ISSUE THREAD

**Issue by rmforster1** [HewlettPackard/POSH-HPEOneView#411]
Please fill in as much information as possible to help resolve your issue.
Connect-HPOVMGMT should display the login message if Acknowledgement is set to $true
### Expected Behavior
The login message should be displayed if it needs to be acknowledged.

### Actual Behavior
only a message to acknowledge is displayed

Please Confirm
Do you acknowledge the login message?
[Y] Yes  [N] No  [?] Help (default is "N"): y

### Steps to reproduce
issue a connect-hpovmgmt without the -LoginAcknowledge parameter and
Set-HPOVLoginMessage has -Acknowledgement $true set

### Version Information
ApplianceVersion: 4.20.01.385523.01 4.20.1971.1960
```PowerShell
Name                           Value
----                           -----
PSVersion                      5.1.14409.1018
PSEdition                      Desktop
PSCompatibleVersions           {1.0, 2.0, 3.0, 4.0...}
BuildVersion                   10.0.14409.1018
CLRVersion                     4.0.30319.42000
WSManStackVersion              3.0
PSRemotingProtocolVersion      2.3
SerializationVersion           1.1.0.1

**Comment by ChrisLynchHPE** [comment]
This is a bug as the API schema changed without me knowing, and the login message is not displayed even if set and enforced.  I'll have it fixed for the next update I'm working on.

**Comment by rmforster1** [comment]
Awesome. I look forward to it. 


Thanks

/Russ

**Comment by ChrisLynchHPE** [comment]
This was fixed in [Release 4.20.2133.3186](https://github.com/HewlettPackard/POSH-HPOneView/releases/tag/v4.20.2133.3186) and in [PowerShell Gallery](https://www.powershellgallery.com/packages/HPOneView.420/4.20.2133.3186).

Apologies for the extremely late reply.

Please verify before this issue is closed.

**Comment by rmforster1** [comment]
This was fixed in 4.20 but has re-appeared in 5.0.

**Comment by ChrisLynchHPE** [comment]
Hmm...  Yep, this is a regression.  Looks like the fix I did for 4.20 didn't merge into 5.00.  I'll have it fixed in the next 5.00 library update.

**Comment by ChrisLynchHPE** [comment]
Closing due to no further activity.  If you wish, feel free to re-open to discuss further.
